QUEST_LV_0100.tsv

```diff
@@ -1033,271 +1033,271 @@ QUEST_LV_0100_20150317_001032	$I should have requested this of you all at once.{
+QUEST_LV_0100_20150317_001136	$I really wonder what has become of this sacred land?
+QUEST_LV_0100_20150317_001137	{memo X}The Caprias were the first to change after the incident four years ago. {nl}And then the Pantos.. now only the babies are left.
+QUEST_LV_0100_20150317_001138	$Our family has been protecting this sacred land for a long time.{nl}Please cooperate with us to keep the reverent atmosphere here.{nl}- Watcher
+QUEST_LV_0100_20150317_001139	$Thank you for helping out. {nl}You've reduced my workload quite a bit.
 QUEST_LV_0100_20150317_001140	{memo X}
-QUEST_LV_0100_20150317_001141	We used to be on good terms with the Pantos. {nl} These days, we're constantly at each other's throats.
+QUEST_LV_0100_20150317_001141	$We used to be on good terms with the Pantos. {nl}These days, we're constantly at each other's throats.
 QUEST_LV_0100_20150317_001142	{memo X}I came back because the elders told me to restore the dolls used for spells.{nl}The dolls for spells. Isn't it exciting?
-QUEST_LV_0100_20150317_001143	Thank you. {nl} We might be able to go get more when more young men arrive.
-QUEST_LV_0100_20150317_001144	{memo X}I can't stand seeing the Panto totems pressed with evil forces. {nl} I'm going to break it through the shaman doll. Want to give it a try?
-QUEST_LV_0100_20150317_001145	$We can't do anything until Basil restores the shaman doll perfectly. {nl} Unless for special situations like this, we can use, um, nothing.
-QUEST_LV_0100_20150317_001146	It would have been a disaster if the Wild Carnivore had legs.{nl}I'm glad it was killed beforehand.
+QUEST_LV_0100_20150317_001143	$Thank you. {nl}We might be able to go get more when more young men arrive.
+QUEST_LV_0100_20150317_001144	{memo X}I can't stand seeing the Panto totems pressed with evil forces. {nl}I'm going to break it through the shaman doll. Want to give it a try?
+QUEST_LV_0100_20150317_001145	$We can't do anything until Basil restores the shaman doll perfectly. {nl}Unless for special situations like this, we can use, um, nothing.
+QUEST_LV_0100_20150317_001146	$It would have been a disaster if the Wild Carnivore had legs.{nl}I'm glad it was killed beforehand.
 QUEST_LV_0100_20150317_001147	{memo X}I heard we promised the first Paladin three things in return for saving our family. {nl}
-QUEST_LV_0100_20150317_001148	$To help build the Tenet Church, {nl} to protect the divine relics by having Watchers guard the sacred land, {nl} and lastly, to protect each other together when great danger arises. {nl}
-QUEST_LV_0100_20150317_001149	I do not know what curse it was. {nl} There's got to be a record within the Tenet Church.
+QUEST_LV_0100_20150317_001148	$To help build the Tenet Church, {nl}to appoint the Watchers to safeguard the sacred land covering the Holy Relic, {nl}and lastly, to protect each other together when great danger arises. {nl}
```

Comment:
In this PR https://github.com/Treeofsavior/EnglishTranslation/pull/1405/files, you named "divine object" to "sacred relic" and you're using the term 'Holy Relic' in this PR.